lib/ci/bin/rails_upgrade_analyzer

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env ruby
+
+require "stringio"
+require_relative "../lib/rails_upgrade_analyzer"
+require_relative "../../deprecation_summary"
+
+module RailsUpgradeAnalyzer
+  class Listener
+
+    REPO_NAME = "Liaison-Intl/WebAdMIT"
+
+    RAILS_4_ACCEPTED = "Rails 4 Accepted"
+    RAILS_4_REJECTED = "Rails 4 Rejected"
+    RAILS_4_REBASE = "Rails 4 CI Needed (rebase master)"
+    RAILS_4_WARNING = "Rails 4 Check Deprecation Warnings"
```

Comment:
What are your thoughts on making these a little more generic?  E.g. so we don't have to change these for Rails 5.